themes/default/content/blog/deploy-ai-ml-vercel-app/index.md

```diff
@@ -0,0 +1,204 @@
+---
+date: 2023-09-11
+title: "Deploying Your AI/ML Chatbot Frontend To Vercel Using Pulumi"
+allow_long_title: true
+meta_desc: "Learn how to effortlessly deploy your AI/ML chatbot frontend to Vercel using Pulumi and Python to take your AI DevOps skills to the next level"
+meta_image: meta.png
+authors:
+    - adora-nwodo
+tags:
+    - ai
+    - ml
+    - iac
+    - vercel
+    - vuejs
+    - chatbot
+
+---
+
+The process of taking an idea and turning it into reality has been nothing short of extraordinary since we started innovating with Artificial Intelligence. With this technology, machines learn about and communicate with people, while also helping us in ways we never could have imagined only a few years ago. If you've been following along, you might recall our [previous article](https://www.pulumi.com/blog/mlops-the-ai-challenge-is-cloud-not-code/) where we used Python and Pulumi to a chatbot API (named katwalk) to the cloud.
+
+But our AI adventure hasn't ended there; it's only just begun.
+
+Now, imagine this: A Vue.js chatbot frontend, seamlessly integrating human interaction and the AI chatbot API. This blog post is part of the series on AI Infrastructure as Python. We'd delve into the creation of a captivating Vue.js chatbot frontend, and with Pulumi, we'll deploy it to Vercel.
```

Comment:
can i trouble you to create just a really simple diagram which shows the arch diagram of all the pieces? 

all the pieces of the backend (compute, the hugging face model on it, how the rest api is being exposed) and then the front end is connected to the baackend